packages/cli/src/info/cdk_info_provider.test.ts

```diff
@@ -0,0 +1,43 @@
+import * as assert from 'node:assert';
+import * as test from 'node:test';
+import { execa } from 'execa';
+import { CdkInfoProvider } from './cdk_info_provider.js';
+
+void test.describe('CDK Info', () => {
+  const mockValue: string = `
+ℹ️ CDK Version: 2.110.1 (build 0d37f0d)
+ℹ️ AWS environment variables:
+  - AWS_SESSION_TOKEN = <redacted>
```

Comment:
we should include all sensitive keys in the test payload and ensure that they are all removed in the output